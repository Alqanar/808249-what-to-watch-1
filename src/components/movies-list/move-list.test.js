import React from "react";
import renderer from "react-test-renderer";

import MovieList from "./movie-list.jsx";

const films = [
  {
    id: `0`,
    name: `The Aftermath`,
    imageLink: `https://st.kp.yandex.net/images/film_iphone/iphone360_1000125.jpg`
  },
  {
    id: `1`,
    name: `The Professor and the Madman`,
    imageLink: `https://st.kp.yandex.net/images/film_iphone/iphone360_996027.jpg`
  },
  {
    id: `2`,
    name: `Five Feet Apart`,
    imageLink: `https://st.kp.yandex.net/images/film_iphone/iphone360_1151373.jpg`
  }
];

it(`Movie list correctly renders`, () => {
  const tree = renderer
    .create(<MovieList
      films={films}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
