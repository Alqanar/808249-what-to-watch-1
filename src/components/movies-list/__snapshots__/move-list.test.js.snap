// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movie list correctly renders 1`] = `
<div
  className="catalog__movies-list"
>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
  >
    <button
      className="small-movie-card__play-btn"
      type="button"
    >
      Play
    </button>
    <div
      className="small-movie-card__image"
    >
      <img
        alt="The Aftermath"
        height="175"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        The Aftermath
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
  >
    <button
      className="small-movie-card__play-btn"
      type="button"
    >
      Play
    </button>
    <div
      className="small-movie-card__image"
    >
      <img
        alt="The Professor and the Madman"
        height="175"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        The Professor and the Madman
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
  >
    <button
      className="small-movie-card__play-btn"
      type="button"
    >
      Play
    </button>
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Five Feet Apart"
        height="175"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Five Feet Apart
      </a>
    </h3>
  </article>
</div>
`;
